### DISCUSSION TITLE
How could I add a Window Store application

### DISCUSSION THREAD

**Discussion by cdjzzzz** [LizardByte/community#514] [Sunshine]
How could  i add a Window Store application into Applicaitons Tab.
Which Cmd or output should i fill in.
Could anyone give me an example?

**Comment by ReenigneArcher** [comment]
In the future, please use https://github.com/orgs/LizardByte/discussions for questions. I converted this for you automatically.

**Comment by ReenigneArcher** [comment]
https://www.bing.com/search?q=start+windows+store+app+from+command+line

**Comment by cathyjf** [comment]
I don't think the feature to add individual applications is all that useful personally, but if the feature should exist, it may make sense to add support for launching UWP apps. It isn't pleasant to launch them from the command line. There is a COM API for launching them. It isn't pleasant either, but at least it would be a one-off thing to add to Sunshine, if somebody wanted to add it.

[a comment by an automated account is not shown]
